wifiphisher/pywifiphisher.py

```diff
@@ -318,14 +327,15 @@ def target_APs():
 
         print ((G + '{0:2}' + W + ' - {1:2}  - ' +
                 T + '{2:{width}} ' + W + ' - ' +
-                B + '{3:17}' + W + ' - {4:12} - ' +
-                R + ' {5:}' + W
+                B + '{3:17}' +  W  + ' - {4:12} - ' +
+                R + ' {5:20}' + B +  ' - {6:3} '
```

Comment:
There are three issues here:

1) The dash (-) should be printed in white, not blue.

2) If the vendor's name is long, it will break the alignment. My recommendation is to only print the first x characters of vendor's name.

3) The column name is "power" but you print the rssi. See here (http://stackoverflow.com/questions/15797920/how-to-convert-wifi-signal-strength-from-quality-percent-to-rssi-dbm) on how to convert the RSSI dBm to Signal Strength Percentage and print the power instead. Furthermore, if the power percentage is more than 50%, we should print it in green, if it's between 30% and 50% in yellow and if it's below 30% in red.